composer.json

```diff
@@ -1,36 +1,54 @@
 {
-  "name": "roots/bedrock-autoloader",
-  "license": "MIT",
-  "description": "An autoloader that enables standard plugins to be required just like must-use plugins",
-  "authors": [
-    {
-      "name": "Nick Fox",
-      "email": "nick@foxaii.com",
-      "homepage": "https://github.com/foxaii"
+    "name": "roots/bedrock-autoloader",
+    "license": "MIT",
+    "description": "An autoloader that enables standard plugins to be required just like must-use plugins",
+    "authors": [
+        {
+            "name": "Nick Fox",
+            "email": "nick@foxaii.com",
+            "homepage": "https://github.com/foxaii"
+        },
+        {
+            "name": "Scott Walkinshaw",
+            "email": "scott.walkinshaw@gmail.com",
+            "homepage": "https://github.com/swalkinshaw"
+        },
+        {
+            "name": "Austin Pray",
+            "email": "austin@austinpray.com",
+            "homepage": "https://github.com/austinpray"
+        }
+    ],
+    "keywords": [
+        "autoloader",
+        "bedrock",
+        "mu-plugin",
+        "must-use",
+        "plugin",
+        "wordpress"
+    ],
+    "support": {
+        "forum": "https://discourse.roots.io/"
     },
-    {
-      "name": "Scott Walkinshaw",
-      "email": "scott.walkinshaw@gmail.com",
-      "homepage": "https://github.com/swalkinshaw"
+    "require": {
+        "php": ">=7.1"
     },
-    {
-      "name": "Austin Pray",
-      "email": "austin@austinpray.com",
-      "homepage": "https://github.com/austinpray"
-    }
-  ],
-  "keywords": [
-    "autoloader", "bedrock", "mu-plugin", "must-use", "plugin", "wordpress"
-  ],
-  "support": {
-    "forum": "https://discourse.roots.io/"
-  },
-  "require": {
-    "php": ">=7.1"
-  },
-  "autoload": {
-    "psr-4": {
-      "Roots\\Bedrock\\": "src/"
+    "autoload": {
+        "psr-4": {
+            "Roots\\Bedrock\\": "src/"
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "2.x-dev"
+        }
+    },
+    "require-dev": {
+        "codeception/codeception": "^4.1",
+        "codeception/module-asserts": "^1.0.0",
+        "codeception/mockery-module": "^0.4.0"
+    },
+    "scripts": {
+      "test": "codecept run"
```

Comment:
We already have a WIP unit test branch here https://github.com/roots/bedrock-autoloader/pull/17